### PR TITLE
doc: Updated Mgr Dashboard documentation

### PR DESCRIPTION
**Pull request by LenzGr**
* Reworded the description of the user/role management feature, clarified
that it can be configured on both the CLI and the WebUI.
* Added labels for referencing the user/role management and SSL configuration sections.
* Added note about the embedding of the Grafana dashboards still being WIP.
* Fixed a typo.

Signed-off-by: Lenz Grimmer <lgrimmer@suse.com>

- [x] Updates documentation if necessary

### CODE REVIEW

**Comment by LenzGr** [comment]
jenkins render docs

**Comment by LenzGr** [comment]
jenkins render docs

File: doc/mgr/dashboard.rst [hunk 1]
```diff
@@ -249,6 +252,13 @@ The default value is 45 seconds.
 Enabling the Embedding of Grafana Dashboards
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+.. note:: 
+  The embedding of Grafana dashboards into the Ceph Manager Dashboard is
+  currently work in progress. This section documents the backend configuration.
+  The corresponding changes to the WebUI have not been merged yet. You can
+  follow the development process in `PR#23666
+  <https://github.com/ceph/ceph/pull/23666>`_.
```
Comment (sebastian-philipp) [inline doc/mgr/dashboard.rst]: I don't know if it makes sense to add these WIP things to the documentation. As soon as #23666 is merged, this note is outdated.

Comment (LenzGr) [reply doc/mgr/dashboard.rst]: > I don't know if it makes sense to add these WIP things to the documentation. As soon as #23666 is merged, this note is outdated.

@sebastian-philipp Agreed, it is somewhat confusing, hence my intention to add a clarifying note to this section.

It was added when the corresponding backend functionality was merged, with the intention of adding the missing frontend code that depends on it later. 

If #23666 obsoletes this part of the documentation, that PR should take care of removing or updating it accordingly.

**Pull request merged**